tickets/src/test/java/no/unit/nva/publication/ticket/create/CreateTicketHandlerTest.java

```diff
@@ -678,6 +678,39 @@ void shouldSetFinalizedByFromRequestUtilsWhenTicketIsAutoApproved()
         assertThat(ticket.getFinalizedBy().toString(), is(equalTo(curatorName)));
     }
 
+    @ParameterizedTest
+    @MethodSource("no.unit.nva.publication.ticket.test.TicketTestUtils#invalidAccessRightForTicketTypeProvider")
+    void shouldNotAllowCuratorWithoutValidAccessRightToCreateTicket(Class<? extends TicketEntry> ticketType,
```

Comment:
What is a curator if they don't have the correct access right?

Reply:
Curator with accessRight not related to ticket they attempt to create, see:


```
    public static Stream<Arguments> invalidAccessRightForTicketTypeProvider() {
        return Stream.of(Arguments.of(DoiRequest.class, AccessRight.MANAGE_PUBLISHING_REQUESTS),
                         Arguments.of(PublishingRequestCase.class, AccessRight.MANAGE_DOI));
                         Arguments.of(GeneralSupportRequest.class, AccessRight.MANAGE_PUBLISHING_REQUESTS));
    }
```

Reply:
It may make more sense if i change Curator to User in test name?

Reply:
I think that was what I was getting at. I find the "X Curator" concepts a bit difficult since they aren't codified